File: Shreya/Arrays and Strings/Valid Anagram.cpp
class Solution {
public:
    bool isAnagram(string s, string t) 
    {
     if (s.length()!=t.length())
	{
		return false;
	}
	int arr[26] = {0};
	for (int i=0;i<s.length();i++)
	{	
		arr[s[i]-97]+=1;
	}
	for (int i=0;i<t.length();i++)
	{	
		arr[t[i]-97]-=1;
	}
	for ( int i = 0; i<26; i++)
	{
		if(arr[i]!=0)
		{
			return false;
		}
	}
	return true;
     
    }
};

File: Utkarsh mishra/Arrays and String/Questions/validanagram.java
class Solution {
    public boolean isAnagram(String s, String t) {
        int b=0;
        if(s.length()!=t.length()){
            return false;}
         int[] a=new int[26];
        for(char c:s.toCharArray()){
            a[c -'a']++;}
        
        for(char c:t.toCharArray()){
           a[c -'a']--;}
        
        for(int i=0;i<26;i++)
        {
            if(a[i]!=0)
                return false;
         }
         
        
    
        return true;
        
        }
}
